src/components/Nodes/GethConfig/VersionList.jsx

```diff
@@ -162,8 +160,12 @@ class VersionList extends Component {
     )
   }
 
-  renderVersionList = () => {
+  isSelectedRelease = release => {
     const { client } = this.props
+    return release.fileName === client.release.fileName
```

Comment:
Breaks on load, because `client.release` is `null`

Reply:
ty